packages/atoms/booker/Booker.tsx

```diff
@@ -0,0 +1,13 @@
+import type { BookerProps } from "@calcom/features/bookings/Booker";
```

Comment:
Any file in `packages/atom` is preparation for the usage of this component as atom. However, to speed up the implementation on our side we decided to first implement the new booker in our own website, before finding solutions for things like dynamic urls, translations, importing next's Link component which we don't want in the atom/package (because that bundles a lot of next as dependency), etc etc. So feel free to ignore this `atoms` directory for now, quite some work left still after the booker itself is merged.